haskell/src/SwiftNav/SBP/Acquisition.hs

```diff
@@ -0,0 +1,26 @@
+module SwiftNav.SBP.Acquisition where
+
+import Data.Int
+import Data.Text
+import Data.Word
+
+
```

Comment:
But not sure about an overall SBP typeclass? Outside of serialization, are there other general methods to define? Also, this isn't covering base message (which will probably not be generated).